src/api.jl

```diff
@@ -429,7 +492,11 @@ function infer(fieldids::Vector{Tuple{Int, Int, Int}},
     # fitting the PSF for the whole catalog above)
     info("fitting PSF for all relevant sources")
     tic()
-    ModelInit.fit_object_psfs!(mp, target_sources, images)
+    if Threaded
+      fit_object_psfs_threaded!(mp, target_sources, images)
+    else
+      ModelInit.fit_object_psfs!(mp, target_sources, images)
+    end
     times.fit_psf = toq()
```

Comment:
Right, and right. We can wait on this until Julia threads are more stable, which will be a good while yet.